src/PieChart.js

```diff
@@ -292,6 +329,35 @@ class PieChart extends React.Component {
     );
   }
 
+  renderSliceLabel(value, slice, center, radius, index) {
+    const {
+      getPieSliceLabel,
+      pieSliceLabelStyle,
+      pieSliceLabelDistance
+    } = this.props;
+    const labelPercent = (slice.end - slice.start) / 2 + slice.start;
+    const style = {
+      textAnchor: "middle",
+      dominantBaseline: "central"
+    };
+
+    if (typeof pieSliceLabelStyle === "function") {
```

Comment:
you might be able to reuse this getValue accessor that we use on other charts https://github.com/spotify/reactochart/blob/master/src/utils/Data.js#L44